src/renderer/components/ft-toggle-switch/ft-toggle-switch.vue

```diff
@@ -15,7 +15,7 @@
       class="switch-input"
       :checked="currentValue"
       :disabled="disabled"
-      @change="$emit('change', currentValue)"
+      @change="change(currentValue)"
```

Comment:
As the event handler accesses `currentValue` itself, you don't have to pass it through.
```suggestion
      @change="change"
```